dcli/lib/src/functions/ask.dart

```diff
@@ -654,23 +657,23 @@ class _AskValidatorValueRange extends AskValidator {
   final num maxValue;
 
   @override
-  String validate(String line) {
+  String validate(String line, String? customErrorMessage) {
     final finalline = line.trim();
 
     final value = num.tryParse(finalline);
     if (value == null) {
-      throw AskValidatorException(red('Must be a number.'));
+      throw AskValidatorException(customErrorMessage ?? red('Must be a number.'));
     }
 
     if (value < minValue) {
       throw AskValidatorException(
-        red('The number must be greater than or equal to $minValue.'),
+        red(customErrorMessage ?? 'The number must be greater than or equal to $minValue.'),
```

Comment:
looks like sometimes we are applying 'red' to the custom message and sometimes not. 
This needs to be consistent.